Applications/TextEditor/TextEditorWidget.h

```diff
@@ -46,6 +46,7 @@ class TextEditorWidget final : public GUI::Widget {
 
     enum class PreviewMode {
         None,
+        Default,
```

Comment:
`Default` is not a preview mode in itself, and so doesn't belong in this enum :)
Please reorganize the code so this state is not required.

Reply:
how about "auto"?
that's really what it is.

Reply:
What I mean is that this shouldn't be a new `PreviewMode` value